.github/workflows/release_gh.yml

```
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
+#
+# SPDX-License-Identifier: MIT
+
+name: GitHub Release Actions
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  upload-release-assets:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Run GitHub Release CI workflow
+      uses: adafruit/workflows-circuitpython-libs/release-gh@main
+      with:
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        upload-url: ${{ github.event.release.upload_url }}
```

I believe you'll need to the additional argument here since it's a package that doesn't start with `adafrruit_`:

```suggestion
        upload-url: ${{ github.event.release.upload_url }}
        package-prefix: jepler_udecimal
```